src/state_title_screen.c

```diff
@@ -102,15 +104,15 @@ void state_init_title_screen(void) {
     gGameStatusPtr->creditsViewportMode = -1;
     intro_logos_update_fade();
     titleData = load_asset_by_name("title_data", &titleDataSize);
-    titleDataDst = D_800A0974 = heap_malloc(titleDataSize);
+    titleDataDst = TitleScreen_ImgList = heap_malloc(titleDataSize);
     decode_yay0(titleData, titleDataDst);
     general_heap_free(titleData);
 
-    D_800A0978 = (s32*)(D_800A0974->unk_00 + (s32) D_800A0974);
-    D_800A097C = (s32*)(D_800A0974->unk_04 + (s32) D_800A0974);
-    D_800A0980 = (s32*)(D_800A0974->unk_08 + (s32) D_800A0974);
+    TitleScreen_ImgList_Logo = (s32*)(TitleScreen_ImgList->logo + (s32) TitleScreen_ImgList);
```

Comment:
Not from your PR, but these pointer types should be IMG_PTR for images and PAL_PTR for palettes instead of s32* for both.

Reply:
This doesn't make sense, as IMG_PTR is a u8* type, but TitleDataStruct uses s32 types

Reply:
we'd want to change the types in that struct, but it's okay to defer this to later since it might have some downstream effects